File: src/main/java/com/qouteall/immersive_portals/ducks/IEVector4f.java
package com.qouteall.immersive_portals.ducks;

import net.minecraft.client.util.math.Vector4f;
import net.minecraft.util.math.Matrix4f;
import org.spongepowered.asm.mixin.Mixin;

//@Mixin(Vector4f.class)
public interface IEVector4f {
    public void transform(Matrix4f matrix) throws NoSuchFieldException, IllegalAccessException;
}

File: src/main/kotlin/com/helixd2s/minevkr/ducks/IEVector4f.kt
package com.helixd2s.minevkr.ducks

import net.minecraft.util.math.Matrix4f

interface IEVector4f {
    @Throws(NoSuchFieldException::class, IllegalAccessException::class)
    fun transform(matrix: Matrix4f)
    fun loadFromArray(arr: FloatArray?)
    fun toArray(): FloatArray
}
